chart/open-feature-operator/README.md

```diff
@@ -82,58 +82,59 @@ helm uninstall open-feature-operator
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
-## Configuration
+## Parameters
```

Comment:
This might break some links (given the HTLM href). 
We could use a config file to let the tool use `Configuration` as a section name: https://github.com/bitnami-labs/readme-generator-for-helm#configuration-file

Reply:
most likely it's because they are trying to import the json file. Try to run it with a fullpath, i.e.:  `readme-generator --config $(pwd)/chart/open-feature-operator/helm-docs-config.json --values=./chart/open-feature-operator/rendered.yaml --readme=./chart/open-feature-operator/README.md`